### DISCUSSION TITLE
Expand devtools sidebar?

### DISCUSSION THREAD

**Discussion by ThinkSalat** [react-hook-form/react-hook-form#4454] [Q&A]
Is there a way to expand it? This is pretty useless if you have medium length names
![image](https://user-images.githubusercontent.com/31082405/111511813-51ffad80-870c-11eb-9109-270f00b912c4.png)

**Comment by bluebill1049** [comment]
there is a feature request, but haven't got chance to work on it, feel free to contribute if you are interested.
